code/controllers/subsystem/traumas.dm

```diff
@@ -122,7 +124,9 @@ SUBSYSTEM_DEF(traumas)
 						/obj/item/clothing/head/helmet/abductor, /obj/structure/bed/abductor, /obj/structure/table_frame/abductor,
 						/obj/structure/table/abductor, /obj/structure/table/optable/abductor, /obj/structure/closet/abductor, /obj/item/organ/heart/gland,
 						/obj/machinery/abductor, /obj/item/crowbar/abductor, /obj/item/screwdriver/abductor, /obj/item/weldingtool/abductor,
-						/obj/item/wirecutters/abductor, /obj/item/wrench/abductor, /obj/item/stack/sheet/mineral/abductor))
+						/obj/item/wirecutters/abductor, /obj/item/wrench/abductor, /obj/item/stack/sheet/mineral/abductor)),
+						"birds" = typecacheof(list(/obj/item/clothing/suit/toggle/owlwings/griffinwings,/obj/item/clothing/shoes/griffin,
```

Comment:
DONT FORGET OWL SUIT

Reply:
I could not find it

Reply:
**/obj/item/clothing/suit/toggle/owlwings**/griffinwings